documentation/deploy_grafana.md

```diff
@@ -248,7 +248,9 @@ spec:
     ...
     envFrom:                        # Environment variables from Secret or ConfigMap. The key of data becomes the env name.
     ...
-    skipCreateAdminAccount: <bool>  # Skip creating the admin account when providing custom credentials from a secret
+    skipCreateAdminAccount: <bool>  # Skip creating the admin account when providing custom credentials from a secret.
+    ...
+    priorityClassName: <string?     # Assign a priorityClass name to the grafana pod. Empty by default.
```

Comment:
<string>

Reply:
typing too damn fast :man_facepalming: ,  fixed in https://github.com/integr8ly/grafana-operator/pull/289/commits/28138606dfaed7aad49d198df8e885af1b8dec2b ,  good spot :+1: